toolkit/geoview-compose/src/main/java/com/arcgismaps/toolkit/geoviewcompose/LocalSceneViewProxy.kt

```diff
@@ -0,0 +1,160 @@
+        set(value) {
+            setGeoView(value)
+            field = value
+        }
+
+    /**
+     * Sets the [localSceneView] parameter on this operator. This should be called by the composable
+     * [LocalSceneView] when it enters the composition and set to null when it is disposed.
+     *
+     * @since 300.0.0
+     */
+    internal fun setLocalSceneView(localSceneView: LocalSceneView?) {
+        this.localSceneView = localSceneView
+    }
+
+    private val nullSceneViewErrorMessage: String =
+        "LocalSceneView must be part of the composition when this member is called."
+
+    /**
+     * Pans or zooms the local scene view using animation to the specified viewpoint location. The
+     * animation takes place over the specified duration. The animation curve defines the animation
+     * easing function.
+     *
+     * @param viewpoint The viewpoint that should be set on the view.
+     * @param durationSeconds The amount of time in seconds to move to the new viewpoint.
+     * @param animationCurve The type of animation curve.
+     * @return a [Result] of true if the viewpoint was successfully set, false otherwise.
+     * @since 300.0.0
+     */
+    public suspend fun setViewpointAnimated(
+        viewpoint: Viewpoint,
+        durationSeconds: Float,
+        animationCurve: AnimationCurve
+    ): Result<Boolean> {
+        return localSceneView?.setViewpointAnimated(
+            viewpoint,
+            durationSeconds,
+            animationCurve
+        ) ?: Result.failure(IllegalStateException(nullGeoViewErrorMessage))
+    }
+
+    /**
+     * Changes the display to the viewpoint specified by the given camera.
+     *
+     * @param camera The camera that should be set on the view.
+     * @since 300.0.0
+     */
+    public fun setViewpointCamera(camera: Camera) {
+        localSceneView?.setViewpointCamera(camera)
+    }
+
+    /**
+     * Pans or zooms the local scene view using animation to the specified camera location.
+     *
+     * @return a [Result] of true if the viewpoint was successfully set, false otherwise.
+     * @since 300.0.0
+     */
+    public suspend fun setViewpointCameraAnimated(
+        camera: Camera
+    ): Result<Boolean> {
+        return localSceneView?.setViewpointCameraAnimated(
+            camera
+        ) ?: Result.failure(IllegalStateException(nullGeoViewErrorMessage))
+    }
+
+    /**
+     * Pans or zooms the local scene view using animation to the specified camera location
+     * asynchronously. Animation takes place over the specified duration.
+     *
+     * @param camera The camera that should be set on the view.
+     * @param duration The amount of time in seconds to move to the new viewpoint.
+     * @return a [Result] of true if the viewpoint was successfully set, false otherwise.
+     * @since 300.0.0
+     */
+    public suspend fun setViewpointCameraAnimated(
+        camera: Camera,
+        duration: Float
```

Comment:
Just noticed that we default `duration` to 0.25 seconds for `SceneViewProxy`. I wonder if we should do the same here and also on the view-based LSV? Could be a follow up issue...

Also, the type should be `Duration`:
```suggestion
        duration: Duration = 0.25.seconds
```

Reply:
Change to duration and defaulted to 3 seconds